type_lens/type_view.py

```diff
@@ -146,3 +146,14 @@ def has_inner_subclass_of(self, cl: type[Any] | tuple[type[Any], ...]) -> bool:
             Whether any of the type's generic args are a subclass of the given type.
         """
         return any(t.is_subclass_of(cl) for t in self.inner_types)
+
+    def strip_optional(self) -> TypeView:
+        if not self.is_optional:
+            return self
+
+        if len(self.args) == 2:
+            return self.inner_types[0]
+
+        args = tuple([a for a in self.args if a is not NoneType])
```

Comment:
```suggestion
        args = tuple(a for a in self.args if a is not NoneType)
```
No need to create the list first.